lis/plugins/LIS_lsmda_pluginMod.F90

```diff
@@ -513,6 +528,8 @@ subroutine LIS_lsmda_plugin
    external noahmp401_scale_veg
    external noahmp401_descale_veg
    external noahmp401_veg_DAlog
+   external noahmp401_transform_veg
```

Comment:
Not sure why this is added, does not seem to be used anywhere. Does no harm.

Reply:
Indeed. Removed this, together with noahmp401_map_veg which is also not used anymore.